schedule/migration/migration_offline_media_all_patterns_x86_64.yaml

```diff
@@ -0,0 +1,38 @@
+name: migration_offline_x86_64.yaml
```

Comment:
This line need to be updated to new name also.

Reply:
Updated to `name: migration_offline_media_all_patterns_x86_64.yaml`